metadata.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "puppetlabs-ruby_task_helper",
+  "version": "0.1.0",
+  "author": "lucy",
```

Comment:
I've tended to put `Puppet, Inc.` as the author: https://github.com/puppetlabs/puppetlabs-python_task_helper/blob/master/metadata.json#L4